browser-test/src/admin/admin_program_image.test.ts

```diff
@@ -19,6 +20,43 @@ test.describe('Admin can manage program image', () => {
     await validateScreenshot(page, 'program-image-none')
   })
 
+  test(
+    'Views program card preview in North Star',
+    {tag: ['@northstar']},
+    async ({page, adminPrograms, adminProgramImage}) => {
+      await test.step('Set up program', async () => {
+        await enableFeatureFlag(page, 'north_star_applicant_ui')
+        await loginAsAdmin(page)
+
+        const programName = 'Test Program'
+        await adminPrograms.addProgram(programName)
+
+        await adminPrograms.goToProgramImagePage(programName)
+      })
+
+      await test.step('Verify preview without image', async () => {
+        await adminProgramImage.expectNoImagePreview()
+        await validateScreenshot(page, 'program-image-none-north-star')
```

Comment:
(question) Do we really need a screenshots? Ideally I'd rather see asserts over screenshots.